### ISSUE TITLE
add new hidaya-theme

### ISSUE THREAD

**Issue by Stunkymonkey** [Komet/MediaElch#1039]
**Is your feature request related to a problem? Please describe.**
I have developed a new theme which.
https://github.com/Stunkymonkey/hidaya

It depends on #1037 & #1033.
So we have to wait for a new release.

**Describe the solution you'd like**
add the theme to the gui

**Additional context**
I have not yet written installation instructions and screenshots. But this will come in the next days.

Anything I could help with to "publish" this theme?

EDIT: related PR https://github.com/mediaelch/mediaelch-doc/pull/12

**Comment by Stunkymonkey** [comment]
or maybe it makes sense, creating a screenshot that is consistent with the previous generated ones

**Comment by bugwelle** [comment]
For it to land in all MediaElch versions, I have to move the `themes.xml` file from kvibes.de to https://github.com/mediaelch/mediaelch-meta . I thought that I had already done it (like the `version.xml`) but it seems that I did not... :-/

**Comment by bugwelle** [comment]
I'm currently working on changing the meta repository for templates. The new meta file can be found here: https://github.com/mediaelch/mediaelch-meta/blob/master/export_themes.xml

Btw, I've updated this section: https://mediaelch.github.io/mediaelch-doc/export/create-theme.html#adjust-the-themes-metadata
Please add your GitHub repository as a `<website>` tag. :-)

**Comment by Stunkymonkey** [comment]
providing the .zip file is something i still have to do.

I will try doing this via github-actions

**Comment by bugwelle** [comment]
Hi, you can simply add a GitHub release. That can be done through the GitHub UI ;-)

I've done so for e.g. https://github.com/mediaelch/mediaelch-theme-digitalrabbit-datatables/releases/tag/v1.0

A release is the best option IMHO. Then I can download the ZIP, verify it, calculate a SHA256 sum and add it to https://github.com/mediaelch/mediaelch-meta/blob/master/export_themes.xml 

The next MediaElch version will do some SHA256 checking. Then I can also link your repository without having to worry about "bad code" (not that I don't trust you, this is just a basic security thing :-) ).

**Comment by Stunkymonkey** [comment]
Thank you very much for all the support. Specially for upgrading the tool https://github.com/Komet/MediaElch/pull/1041

Adding the shasum is a good idea.

PR at: https://github.com/mediaelch/mediaelch-meta/pull/2

**Comment by bugwelle** [comment]
The next nightly version will use the new meta repository. I'll close this issue and we can move the discussion there. 😃 

"Fixed" by #1041

**Comment by bugwelle** [comment]
The latest nightly will now list your theme as well. Thanks! 🎉

**Comment by Stunkymonkey** [comment]
Thanks a lot for all your work putting into this tool.:heart:
